README.md

```diff
@@ -1,2 +1,125 @@
-# nbde_client
-Ansible role for configuring Network Bound Disk Encryption clients (e.g. clevis)
+nbde_client
+-----------
+
+Ansible role for configuring Network-Bound Disk Encryption clients (e.g. clevis).
+
+This role currently supports `clevis` as a provider and it uses it for operations like encryption
+and decryption.
+
+
+Supported Distributions
+-----------------------
+* RHEL-7+, CentOS-7+
+* Fedora
+
+
+Limitations
+-----------
+This role can currently create `tang` bindings. TPM2 is not supported as of now.
+
+
+Role Variables
+--------------
+
+These are the variables that can be passed to the role:
+
+
+| **Variable** | **Default/Choices** | **Description** |
+|----------|-------------|------|
+| `nbde_client_provider` | `clevis`| identifies the provider for the `nbde_client` role. We currently support `clevis`.|
+| `nbde_client_bindings` | | a list containing groups of LUKS devices and applicable settings for a binding. This group of devices contains specific binding configurations to be applied to the set of devices.
+|`nbde_client_remote_dir` | `/root/.local/share/nbde_client/` |  specifies a directory in the remote hosts that may be used for storing temporary data such as transferred keyfiles. |
```

Comment:
What is a "remote host"? Is this the managed host?
If the directory is used only for temporary data, is there a need for it to be configurable?

Reply:
Yeah, the managed host. And you are right, there is no real need for this to be configured.

Reply:
Can you please remove the variable then? We should not provide knobs that won't be useful.

Reply:
Sure, I will fiix this.